client/src/partials/patient_edit/edit_patient.js

```diff
@@ -0,0 +1,269 @@
+angular.module('bhima.controllers')
+.controller('patientEdit', [
+  '$scope',
+  '$routeParams',  
+  '$translate',
+  'connect',
+  'validate',
+  'messenger',
+  'util',
+  'appstate',
+  function ($scope, $routeParams, $translate, connect, validate, messenger, util, appstate) {
+    var dependencies = {},
+        original_patient_data = null,
```

Comment:
For JS variables, we prefer to us mixedCase.  Unfortunately, the database is constructed using underlines_not_spaces syntax, so we cannot get away from the underlines in some cases.  Could you change `original_patient_data` and `editable_fields`, etc, to mixedCase?